webpack.config.ts

```diff
@@ -1,24 +1,137 @@
+import CopyWebpackPlugin from 'copy-webpack-plugin';
```

Comment:
if this config using only for instrumented
then let's rename i webpack.config.instrumented.js